nx_assets/readme.txt

```diff
@@ -0,0 +1 @@
+nx_assets got here.
```

Comment:
If goal is to keep this folder just `touch .gitignore`

Reply:
If you don't want to commit any of the files use the following in
`nx_assets/.gitignore`:
```
# Ignore everything in this directory
*
# Except this file
!.gitignore
```